### ISSUE TITLE
commands not found

### ISSUE THREAD

**Issue by justaquestionfromme** [wmorellato/babel#13]
Hi!
I freshly installed the extension but it seems broken.
I re-installed the extension multiple times, reloaded VS Code, restarted it, even restarted my PC, but nothing changes.
I opened a folder with Code, it asked me if I wanted to target the folder for my stories and I pressed 'Yes'. Still, nothing has changed. When I press the "Add Story" text, nothing happens. When I press the +, or the Refresh icon, it says "command 'babel.refreshExplorer' not found" or "command 'babel.newStory' not found"

If you need any logs or something, tell me where can I find them and I'll paste them asap.

**Comment by wmorellato** [comment]
Hello! Thanks for using Babel. Please, send me the console output so I can see what's wrong. Do this:
1. first clean up your folder (remove babel.json and any other files in there)
2. open vscode, go to Help > Toggle Developer Tools (at the bottom)
3. in the new window that shows up, go to Console and first clean up the logs (Ctrl+L)
4. now start Babel, select Yes to use the current folder and then try to create a new story

The error should pop up in the console, so just copy it and paste it to me here. I'll see what I can find. 
Apart from this, what system are you using? And also, what's the directory are you trying to use? (make sure to remove any username or personal info from the path before pasting, please)

**Comment by justaquestionfromme** [comment]
Is there a way to get the logs from when I posted the issue?
I did what you wrote me to do and it seems like it's working now. Idk if the problem solved itself or VS Code was just bugging around, but if I'll get the same error I'll definitely comment them later (with the logs).

I'm using Windows 10 (10.0, build: 19042) and tried to access the folder: C:\Users\[username]\Documents\pp\story

**Comment by justaquestionfromme** [comment]
Hello again!
It happened again, this time I caught the logs!
https://pastebin.com/qcZfjBRU
Also, there is a screenshot of the logs!
https://imgur.com/a/zCa7o9x

**Comment by justaquestionfromme** [comment]
_`((I did not censor my username and stuff since it's the same as my GitHub username))`_

**Comment by wmorellato** [comment]
Thanks for the update and for the info! That's really odd, though. I'll try to investigate this on this weekend and if I find a fix I'll push a patch. I'll keep you informed.

**Comment by wmorellato** [comment]
So, no luck. In short, this error is happening because Babel is not being loaded for some reason, which should be in the logs. For example, I am able to get the same error if I leave open the file of a version that I had already deleted in the past. The record for this story is no longer present in the database (`babel.json` file), but since I have checked the option to not hard delete any files, it's still present in the workspace folder:

![Screenshot from 2021-05-29 13-20-09](https://user-images.githubusercontent.com/58891430/120077603-9949d900-c081-11eb-8f59-966b585c426b.png)

This is probably not what is happening with you, but the final result is the same, the extension is not loaded and then none of its commands is available.

What I suggest you is:
1. I just pushed a new version fixing a silly thing that I did on 1.5.0. I reinstalled every dependency and recompiled everything. Install this version and see if it works.
2. I uploaded [here](https://github.com/wmorellato/babel/releases/download/1.5.1/babel-1.5.1-debug.vsix) an unminified version of 1.5.1, you could try to install this one and see if the logs in the Developer Console gets clearer.
3. As it appears, the root cause for this if VSCode not being able to activate Babel. Maybe we missed this error message in the Console, or maybe it was cleared before you could see. Try to open VSCode first (don't load any folder yet), open the Developer Console and then open your Babel directory through VSCode. It will try to activate the extension automatically, see if there's some activation error in the Console.
4. Did you try to update VSCode?

**Comment by justaquestionfromme** [comment]
Yes, I missed that warning somehow!
Anyway, it was lucky for me! So, what I did was:
1. uninstall _all_ the extensions
2. install the ones I already had (let's say A, B, and C)
3. then install Babel (from the vsix you uploaded)
After that, I tried opening a completely empty folder and make a new story, and it was working well.
When I tried to open an older (like 5 days ago on my comment about "the code fixed itself") story, it said "workbench.desktop.main.js:735 Not a Babel workspace! Open VSCode on a Babel directory and try again.". I just moved to an empty folder and copy-pasted my story from the older one to the fresh one, and everything works fine.
I restarted, updated, reloaded everything many times, never seen that bug again. 

When I installed from the vsix, it pushed the warning below, then successfully installed the extension. Also, when I load up Code, it pops an error every time like that: https://pastebin.com/9PBujE53
Not sure if it's Babel or not, but I'll leave them there in case it is.

After all, it works for me now! I reloaded, re-opened Code, Babel, and different random stories I made, and it's still working. 

About updating VSCode, yes, I check for updates almost every day.
